detox/src/errors/DetoxRuntimeError.js

```diff
@@ -1,14 +1,15 @@
+const _ = require('lodash');
+
 class DetoxRuntimeError extends Error {
   constructor({ message = '', hint = '', debugInfo = '' } = {}) {
-    super(`DetoxRuntimeError: ${message}` +
-      (!hint ? '' : '\n\nHINT: ' + hint) +
-      (!debugInfo ? '' : '\n\n' + debugInfo));
-
-    Error.captureStackTrace(this, DetoxRuntimeError);
-  }
+    const formattedMessage = _.compact([
+      message,
+      hint && `HINT: ${hint}`,
+      debugInfo
+    ]).join('\n\n');
 
-  toString() {
-    return super.toString().replace(/^Error: /, '\n');
+    super(formattedMessage);
+    this.name = 'DetoxRuntimeError';
```

Comment:
This is an idiomatically correct way to extend `Error` (using `this.name =`). So, I fixed DetoxRuntimeError implementation.